integration-tests/joyplot/index.js

```diff
@@ -0,0 +1,91 @@
+"use strict"
+
```

Comment:
Instead of creating a build for the original source I included a transpiled version of https://codepen.io/emeeks/pen/EoqRLa